src/design-system/components/AnimatedRoute/AnimatedRoute.tsx

```diff
@@ -221,7 +218,7 @@ export const AnimatedRoute = forwardRef((props: AnimatedRouteProps, ref) => {
   const containerRef = useRef<HTMLDivElement>(null);
 
   const content = (
-    <AccentColorProviderWrapper
+    <AccentColorProvider
```

Comment:
i'd maintain the name `AccentColorProviderWrapper` since `AccentColorProvider` already exists in design-system

Reply:
actually nvm looks good